lib/routes/account.js

```diff
@@ -336,14 +340,17 @@ module.exports = function (
         var email = form.email
         var authPW = Buffer(form.authPW, 'hex')
         var service = request.payload.service || request.query.service
+        var redirectTo = request.payload.redirectTo || request.query.redirectTo
```

Comment:
`redirectTo` and `resume` need to be added to the payload validation on ~ 309. I don't think these should ever come in from the query params, the other endpoints that accept the two fields only accept them from POST data.